source/MaterialXTest/ShaderValid.cpp

```diff
@@ -652,6 +667,9 @@ TEST_CASE("MaterialX documents", "[shadervalid]")
     // This will be imported in each test document below
     mx::DocumentPtr dependLib = mx::createDocument();
     loadLibraries({ "stdlib", "sxpbrlib" }, searchPath, dependLib);
+#ifdef MATERIALX_BUILD_GEN_GLSL
+    glslColorManagementSystem->loadLibrary(dependLib);
```

Comment:
It seems the CMS is loading the complete stdandard libraries here? Only the CM implementations would need to be loaded right?

Reply:
This is true, but it just seemed easier this way. Otherwise I'll need to load another document.

Reply:
My point is just that searching _all_ MaterialX stdlib+sxpbrlib elements when it's really just a few CM implementations you need to consider seems unefficient. But we can improve on this later as well.